zokrates_core/src/ir/from_flat.rs

```diff
@@ -67,7 +67,10 @@ impl<T: Field> From<FlatProg<T>> for Prog<T> {
         // get the main function
         let main = flat_prog.main;
 
-        // get the interface of the program, ie which inputs are private and public
+        // get the signature to keep high level information in the low level representation
```

Comment:
This change should be removed, I suspect merging develop into your branch should work.

Reply:
Fixed at e1c5b6cd422ca24f728bfc58441ff6a41f387c35